src/func.rs

```diff
@@ -230,20 +233,16 @@ impl From<Trap> for ResumableError {
 }
 
 /// A resumable invocation handle. This struct is returned by `FuncInstance::invoke_resumable`.
-pub struct FuncInvocation<'args> {
-	kind: FuncInvocationKind<'args>,
+pub struct FuncInvocation {
+	kind: FuncInvocationKind,
 }
 
-enum FuncInvocationKind<'args> {
+enum FuncInvocationKind {
```

Comment:
It doesn't look like we were using the `args`, @pepyakin what was the intention here?